src/LinuxTracing/TracerImpl.cpp

```diff
@@ -1355,16 +1361,21 @@ uint64_t TracerImpl::ProcessSampleEventAndReturnTimestamp(const perf_event_heade
     DeferEvent(event);
 
   } else if (is_sched_switch_with_callchain) {
-    // TODO(mahmooddarwish): the implementation of this case will be implemented later
+    // TODO(b/243510000): the implementation of this case will be added later
 
   } else if (is_sched_wakeup_with_callchain) {
-    // TODO(mahmooddarwish): the implementation of this case will be implemented later
+    // TODO(b/243510000): the implementation of this case will be added later
 
   } else if (is_sched_switch_with_stack) {
-    // TODO(mahmooddarwish): the implementation of this case will be implemented later
+    // TODO(b/245529464): Avoid copying the stack if the record is from another process
```

Comment:
This seems to be quite important, let's not lose track.

Reply:
It is. #4148 has a potential implementation. I am currently extracting the pieces from this big prototype PR and sending individual PRs.